packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -896,14 +898,56 @@ const Conversation = WebexPlugin.extend({
         async: false
       });
     }
+    /**
+   * helper function to cloneDeepWith for copying instance function
+   * @param {T} value
```

Comment:
What type of object is `{T}`? It should be a javascript type or something defined in a `@typedef` statement.

Reply:
T can be of any type string, object, Symbol etc. Let me see if there is other representation for generic datatype.